### PR TITLE
Add mostly-placeholder ECT dashboard

### PR DESCRIPTION
**Pull request by callumacrae**
### Context

https://dfedigital.atlassian.net/browse/CPDEL-239

### Testing

None yet as it's mostly placeholder stuff so functionality can't be tested yet

### CODE REVIEW

File: app/views/dashboard/show.html.erb [hunk 1]
```diff
@@ -1,35 +1,42 @@
...
-      <h2 class="govuk-heading-l">Your list of mentees</h2>
-      <% current_user.mentor_profile.early_career_teachers.each do |early_career_teacher| %>
-        <p><%= early_career_teacher.full_name %></p>
-      <% end %>
-      <% if current_user.mentor_profile.early_career_teachers.empty? %>
-        <p>You have no mentees</p>
-      <% end %>
-    <% end %>
+    <div class="blue-header govuk-!-padding-left-3 govuk-!-padding-right-3 govuk-!-padding-top-6 govuk-!-padding-bottom-9 govuk-!-margin-bottom-6">
+      <h1 class="govuk-heading-xl govuk-!-margin-bottom-1 blue-header__title">Welcome Joe</h1>
```
Comment (callumacrae) [inline app/views/dashboard/show.html.erb:4]: Considering moving all the spacing stuff into the SCSS for `blue-header`, does anyone else have any thoughts on this?

Comment (kicferk1) [reply app/views/dashboard/show.html.erb:4]: Yes please!

File: app/views/dashboard/show.html.erb [hunk 2]
```diff
@@ -1,35 +1,42 @@
...
+        </div>
+      </div>
+      <div class="govuk-grid-column-one-third-from-desktop">
+        <div style="background-color:#F3F2F1; padding: 10px;">
+          <h1 class="govuk-heading-xl govuk-!-margin-bottom-1">8</h1>
+          <p class="govuk-!-margin-bottom-1">Modules to complete</p>
+        </div>
+      </div>
+    </div>
+    -->
```
Comment (kicferk1) [inline app/views/dashboard/show.html.erb:31]: Just kill this :)

Or, if you think we want to keep them, make them work. We can figure out the progress for modules and lessons in total, although it's a bit harder, so maybe it should be a separate ticket.

File: app/views/dashboard/show.html.erb [hunk 3]
```diff
@@ -1,35 +1,42 @@
...
+
+    <%= govuk_link_to "Go to your module materials", cip_index_path, button: true %>
```
Comment (kicferk1) [inline app/views/dashboard/show.html.erb:33]: I want a specific cip here - hide it behind `<% if current_user.early_career_teacher? %>` and add a link to
```
<%= govuk_link_to "Go to your module materials", cip_path(current_user.early_career_teacher_profile.core_indiction_programme), button: true %>
```

Comment (kicferk1) [reply app/views/dashboard/show.html.erb:33]: In fact, we might want to hide most of this stuff behind that early career teacher check.

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -54,9 +57,6 @@
                 <%= govuk_link_to "Login", new_user_session_path, class: "govuk-header__link" %>
               </li>
               <% end %>
-              <li class="govuk-header__navigation-item govuk-header__navigation-item--active">
-                <%= govuk_link_to "Core Induction Programme", cip_index_path, class: "govuk-header__link" %>
-              </li>
```
Comment (kicferk1) [inline app/views/layouts/application.html.erb:59]: Can we keep it for admin users please?

File: app/views/dashboard/show.html.erb (hunk 3 above)
Comment (callumacrae) [reply app/views/dashboard/show.html.erb:33]: I'd imagine that we'd want an entirely separate dashboard with its own controller for non-ECTs, right?

Comment (kicferk1) [reply app/views/dashboard/show.html.erb:33]: I hate this conversation.

Sooner or later maybe we do, for time being let's keep it in one place.

**Review by kicferk1 — APPROVED** [review]

File: spec/cypress/integration/accessibility.js [hunk 5]
```diff
@@ -4,7 +4,7 @@ describe("Accessibility", () => {
     cy.checkA11y();
   });
 
-  it("Login should be accessible", () => {
+  it.only("Login should be accessible", () => {
```
Comment (kicferk1) [inline spec/cypress/integration/accessibility.js:7]: I don't think we want to keep this change ;) Nor the comment on line 40.

Comment (callumacrae) [reply spec/cypress/integration/accessibility.js]: Line 40 was intentional - it'll be back when the preferred name stuff is done

Comment (kicferk1) [reply spec/cypress/integration/accessibility.js]: Ah, alright.

**Pull request merged**